aws/Makefile

```diff
@@ -31,22 +31,3 @@ terminator_lambda:
 .PHONY: test_policy
 test_policy:
 	$(RUN_TEST_POLICY_PLAYBOOK)
-
```

Comment:
It looks like `VAULT_ARGS` above is unused. Also, the "Running Tests" section in the help above is obsolete.

Alternatively, what do you think of converting the rest of this `Makefile` over to running through `tox`?

Reply:
I've done the clean up. It would be nice indeed to move everything to `tox`. But I would prefer to do it later in another PR later once the Ubuntu22.04/Python3.9 upgrade is done.